doc/getting-started/quick_start.ipynb

```diff
@@ -335,28 +429,100 @@
+    "DeePMD-kit requires a `json` format file to specify parameters for training. "
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 8,
-   "id": "f0b2fa6b-81c2-41ec-b29e-37ff1f6e3b31",
+   "execution_count": 10,
+   "id": "d371a240",
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "{\n",
+      "    \"_comment\": \" model parameters\",\n",
+      "    \"model\": {\n",
+      "\t\"type_map\":\t[\"H\", \"C\"],\n",
+      "\t\"descriptor\" :{\n",
+      "\t    \"type\":\t\t\"se_e2_a\",\n",
+      "\t    \"sel\":\t\t\"auto\",\n",
+      "\t    \"rcut_smth\":\t0.50,\n",
+      "\t    \"rcut\":\t\t6.00,\n",
+      "\t    \"neuron\":\t\t[25, 50, 100],\n",
+      "\t    \"resnet_dt\":\tfalse,\n",
+      "\t    \"axis_neuron\":\t16,\n",
+      "\t    \"seed\":\t\t1,\n",
+      "\t    \"_comment\":\t\t\" that's all\"\n",
+      "\t},\n",
+      "\t\"fitting_net\" : {\n",
+      "\t    \"neuron\":\t\t[240, 240, 240],\n",
+      "\t    \"resnet_dt\":\ttrue,\n",
+      "\t    \"seed\":\t\t1,\n",
+      "\t    \"_comment\":\t\t\" that's all\"\n",
+      "\t},\n",
+      "\t\"_comment\":\t\" that's all\"\n",
+      "    },\n",
+      "\n",
+      "    \"learning_rate\" :{\n",
+      "\t\"type\":\t\t\"exp\",\n",
+      "\t\"decay_steps\":\t50,\n",
+      "\t\"start_lr\":\t0.001,\t\n",
+      "\t\"stop_lr\":\t3.51e-8,\n",
+      "\t\"_comment\":\t\"that's all\"\n",
+      "    },\n",
+      "\n",
+      "    \"loss\" :{\n",
+      "\t\"type\":\t\t\"ener\",\n",
+      "\t\"start_pref_e\":\t0.02,\n",
+      "\t\"limit_pref_e\":\t1,\n",
+      "\t\"start_pref_f\":\t1000,\n",
+      "\t\"limit_pref_f\":\t1,\n",
+      "\t\"start_pref_v\":\t0,\n",
+      "\t\"limit_pref_v\":\t0,\n",
+      "\t\"_comment\":\t\" that's all\"\n",
+      "    },\n",
+      "\n",
+      "    \"training\" : {\n",
+      "\t\"training_data\": {\n",
+      "\t    \"systems\":     [\"../00.data/training_data\"],\n",
+      "\t    \"batch_size\":  \"auto\",\n",
+      "\t    \"_comment\":\t   \"that's all\"\n",
+      "\t},\n",
+      "\t\"validation_data\":{\n",
+      "\t    \"systems\":\t   [\"../00.data/validation_data\"],\n",
+      "\t    \"batch_size\":  \"auto\",\n",
+      "\t    \"numb_btch\":   1,\n",
+      "\t    \"_comment\":\t   \"that's all\"\n",
+      "\t},\n",
+      "\t\"numb_steps\":\t10000,\n",
+      "\t\"seed\":\t\t10,\n",
+      "\t\"disp_file\":\t\"lcurve.out\",\n",
+      "\t\"disp_freq\":\t200,\n",
+      "\t\"save_freq\":\t1000,\n",
+      "\t\"_comment\":\t\"that's all\"\n",
+      "    },    \n",
+      "\n",
+      "    \"_comment\":\t\t\"that's all\"\n",
+      "}\n",
+      "\n"
+     ]
+    }
+   ],
    "source": [
-    "os.chdir(\n",
-    "    os.path.join(prefix_path, \"colombo-academy-tutorials\", \"DeePMD-kit\", \"01.train\")\n",
-    ")"
+    "! cd DeePMD-kit_Tutorial/01.train/ && cat input.json"
```

Comment:
We may use the [new dargs feature](https://docs.deepmodeling.com/projects/dargs/en/latest/nb.html) to show the script.